app/hotels/src/allHotels/__tests__/__snapshots__/AllHotelsSearchList.test.js.snap

```diff
@@ -13,7 +13,7 @@ exports[`renders a "not found" message when no hotel is found 1`] = `
 `;
 
 exports[`renders found hotels 1`] = `
-<View>
+<Component>
```

Comment:
@tbergq Do you know what is behind this change?

https://github.com/react-native-community/react-native-releases/blob/master/CHANGELOG.md#055